mhs/inbound/main.py

```diff
@@ -62,6 +66,7 @@ def start_inbound_server(certs_file: str, key_file: str, party_key: str,
                          ) -> None:
```

Comment:
The type hint of the line above this looks wrong, it looks like it's saying that `persistence_store` is a module instead of a class instance.